v2/pkg/stratus/platform.go

```diff
@@ -28,3 +28,24 @@ func PlatformFromString(name string) (Platform, error) {
 		return "", errors.New("unknown platform: " + name)
 	}
 }
+
+func (p Platform) FormatName() (string, error) {
```

Comment:
Moved the `formatPlatformName` function here so it could be used for the `MarshalYAML` method as well.